build.gradle

```diff
@@ -159,14 +159,14 @@ sonarqube {
         property "sonar.projectName", "am-role-assignment-batch-service"
         property "sonar.projectKey", "am-role-assignment-batch-service"
         property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/reports/jacoco/test/jacocoTestReport.xml"
+        property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/roleassignmentbatch/*RoleAssignmentBatchApplication.java"
```

Comment:
I believe wild card * is not required here as we will be excluding only a particular class.

Reply:
Its update for now and will update exclusions later while working on code coverage.